gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -1118,6 +1119,20 @@ async fn test_gateway_configuration() -> anyhow::Result<()> {
     })
     .await;
 
+    // Verify that the gateway's password is set and its the hash of the plaintext
```

Comment:
Nit: I think this comment is somewhat unclear. What about:

"Verify that the gateway's password is stored correctly (i.e. the stored hash and salt match the password)"